docs/core/concepts/results.md

```diff
@@ -89,12 +89,16 @@ def my_task():
 
 If checkpointing is turned on, the Prefect pipeline will persist the return value of tasks using the `write` method of the task's configured `Result` subclass. Consider the following example task configured to store its return value as a `LocalResult` into the `~/Desktop/HelloWorld/results` directory.
 
+::: tip Enabling Checkpointing During Local Testing
+Checkpointing is only turned on by default when running on Prefect Cloud or Server. To enable checkpointing for local testing, set the `PREFECT__FLOWS__CHECKPOINTING` environment variable to `true`.
+:::
+
 ```python
 from prefect import Flow, task
 from prefect.engine.results import LocalResult
 
 @task
-def my_task(checkpointing=True, result=LocalResult(dir='~/Desktop/HelloWorld/results')):
+def my_task(result=LocalResult(dir='~/Desktop/HelloWorld/results'), target='result.bytes'):
```

Comment:
So I actually don't think we should use the `target` kwarg in this example - `target`s are for caching, i.e., that this task will run only once (and otherwise use the data in the written file), whereas checkpointing is purely for 1.) recovery from failure and 2.) being able to inspect data midway through a flow

Reply:
Gotcha. Didn't know about that distinction. Will remove!

Reply:
FYI @JacksonMaxfield you can still hardcode a filepath to the result using `location`, e.g.,
```python
LocalResult(dir='~/Desktop/HelloWorld/results', location='result.bytes')
```

Reply:
Ahhh interesting. So assuming then that if someone runs the flow again, that location is overwritten with new results (if they happened to change)? `target` -> caching location, `location` -> simply a storage only location

Reply:
That's exactly right, and _both_ target and location can be templated / customized using the same syntax.